### PR TITLE
Connection details UI & Scan Code UI created

### PR DESCRIPTION
**Pull request by Pan-LLi**

### CODE REVIEW

File: front-end/src/ConnectionDetails.js
Comment (XinranTaoAngela) [inline front-end/src/ConnectionDetails.js:1]: make sure to add the header and footer to this page

File: front-end/src/ScanCode.js
Comment (XinranTaoAngela) [inline front-end/src/ScanCode.js:1]: For the image, we need to generate from Mockaroo with axios. Also, is this page for users to scan code like use camera? I'm wondering if we will add the camera functionality later or do we set it up now.

**Review by XinranTaoAngela — CHANGES_REQUESTED** [review]
Over all pretty good! please review my comments and see if theres anything unclear, free to discuss! also, you can use eslint to generate a default formatting so all of our code format the same.

File: front-end/src/ScanCode.js
Comment (Pan-LLi) [reply front-end/src/ScanCode.js:1]: We can add the camera function later, because the qr code information is related to our database

Comment (XinranTaoAngela) [reply front-end/src/ScanCode.js:1]: got it

**Pull request merged**